docs/plugins/markdown/prismjs.md

```diff
@@ -194,6 +194,207 @@ export default defineUserConfig({
 })
 ```
 
+### collapsedLines
+
+- Type: `boolean | number | 'disabled'`
+
+- Default: `'disabled'`
+
+- Details: Whether to enable code block collapsing.
```

Comment:
I think this could be changed to `Default behavior of code block collapsing`

And the description of the first two settings could be added with the words `by default`